Remove IsTest=true from some utility projects

mconnew opened the pull request:
This is blocking updating of versions pulled in by Aracade

HongGit commented:
@mconnew why remove one, the other just change value from true to false?  why not remove both?

HongGit commented on src/System.Private.ServiceModel/tests/Common/Infrastructure/Infrastructure.Common.csproj:
```diff
@@ -3,7 +3,7 @@
     <OutputType>Library</OutputType>
     <CLSCompliant>false</CLSCompliant>
-    <IsTestProject>true</IsTestProject>
+    <IsPackable>false</IsPackable>
```
Why not remove this line as well for consistency?

HongGit replied:
Huh, IsTest is removed, but add IsPackable

mconnew commented:
I think you might have misread the diff. I did remove it from both. I also added IsPackable=false to the Infrastructure project as it was in the unit test project. I wanted to make sure that removing the IsTest property didn't cause the project to attempt to be packaged. Rather than spend time experimenting, as it's already in the Unit project, I just added it to the infrastructure one too.
